rclpy/rclpy/topic_endpoint_info.py

```diff
@@ -166,9 +166,9 @@ def __str__(self):
         result += 'QoS profile:\n'
         result += '  Reliability: %s\n' % self.qos_profile.reliability.name
         result += '  Durability: %s\n' % self.qos_profile.durability.name
-        result += '  Lifespan: %d nanoseconds\n' % self.qos_profile.lifespan.nanoseconds
-        result += '  Deadline: %d nanoseconds\n' % self.qos_profile.deadline.nanoseconds
+        result += '  Lifespan: %s\n' % str(self.qos_profile.lifespan)
+        result += '  Deadline: %s\n' % str(self.qos_profile.deadline)
```

Comment:
feel free to ignore, but it would be much nicer to use f-strings

Reply:
I was feeling that if I update one to f-strings, I should update them all at the same time, but wasn't sure if that was an overreach on the scope of this PR - what do you think?

Reply:
> what do you think?

sounds fine to me to do it here